src/composables/balances.ts

```diff
@@ -70,7 +71,7 @@ export function useBalances({ store }: IDefaultComposableOptions) {
   async function updateBalances() {
     const sdk = await getSdk();
     const balancesPromises = accounts.value.map(
-      ({ address }) => sdk.balance(address).catch((error) => {
+      ({ address }) => sdk.getBalance(address as Encoded.AccountAddress).catch((error) => {
```

Comment:
Please add `TODO` before this that would inform that the `IAccount` should be changed to use the `Encoded.AccountAddress` and then this casting should be removed.
Or maybe you want to do it now? :) But this would require massive changes in many places and as far as I remember Nikita's PR is actually changing this.

Reply:
I think there's no point in doubling the work then